man/fi_efa.7.md

```diff
@@ -103,6 +103,20 @@ provider for AWS Neuron or Habana SynapseAI.
   emulating Read, Write, and Atomic operations (return value is true), or if
   these operations are assisted by hardware support (return value is false).
 
+*FI_OPT_EFA_SENDRECV_IN_ORDER_ALIGNED_128_BYTES - bool*
+: This option only apply to the fi_setopt() call.
+  It is used to force the endpoint to use in-order send/recv operation for each 128 bytes
+  aligned block. Enabling the option will guarantee data inside each 128 bytes
+  aligned block being sent and received in order. If endpoint is not able to
```

Comment:
Questions:
 - Does setting this option apply to local receives or remote deliveries?  is the flag required on both ends?
 - is the alignment also 128 bytes?